pom.xml

```
@@ -97,11 +97,11 @@
     <distributionManagement>
         <snapshotRepository>
             <id>ossrh</id>
```

This should also be changed
```suggestion
            <id>payara-nexus-enterprise-snapshots</id>
```